Adding profile for new student

yenibel opened the pull request.

yenibel committed:
Update authors.yml

yenibel committed:
Archive Critical Review of Gabriel García Márquez archives

I think that I'm doing something wrong with the Markdown format. It seems like it doesn't work.  I follow the tips, but I'm not sure. I have a problem to find the question mark for the beginning of the question. In Spanish we need it. Anyway. This is for far and sure my first draft. Thank you in advance.

halperta committed:
Merge branch 'gh-pages' into gh-pages

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+ 
```
Please add a YAML header: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+ 
+#Gabriel García Márquez. Harry Ransom Center. The University of Texas. Web.16 de Octubre 2018.https:[//hrc.contentdm.oclc.org/digital/collection/p15878coll51/]
```
Please correct the formatting by adding spaces after the "3" and each period.

halperta replied:
Please follow markdown formatting for the url: put the entire URL in square brackets [ and add the url again in parentheses to create a link.

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+#Gabriel García Márquez. Harry Ransom Center. The University of Texas. Web.16 de Octubre 2018.https:[//hrc.contentdm.oclc.org/digital/collection/p15878coll51/]
+
+
+Así de extraño como el nacimiento del último de los Buendía que nació con una cola de cerdo en la famosa novela 100 años de Soledad del colombiano Gabriel García Márquez, es extraño para muchos de sus paisanos que el archivo privado de este escritor, premio Nobel de literatura 1982, reposen en el Centro de Archivos Harry Ransom Center en la Universidad de Texas en Austin, en los Estados Unidos. P ...
```
Use markdown formatting to put the book title in italics. (*)

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+...Uno de sus hijos, Rodrigo García, declaró en una ocasión, que la familia, madre e hijos siempre sintieron el compromiso de hacer que el archivo de su padre llegará a la mayor audiencia posible. En este sentido, el proyecto de digitalizar el archivo por parte de la  reconocida institución Harry Ransom Center parecía lo más cercano a este deseo...
+
+
+...El archivo digitalizado de Gabriel García Márquez abarca más de 27.000 imágenes entre sus documentos y fotografías. Esta colección accesible a todo público mediante la web incluyen manuscritos originales de muchas de sus obras como “100 años de Soledad” y “El Coronel No tiene quién le Escriba”. También una parte de sus memorias escritas en la obra “En Agosto nos Vemos”, que el escritor nunca c ...
```
Incluye?

halperta replied:
I find myself wondering: How significant is "el resto  de esta obra"? How much of the collection is available online?

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+
+En la colección digital se encuentra fotografías que muestran las relaciones del escritor, desde sus encuentros con amigos, también famosos, como Fidel Castro hasta sus relaciones en la intimidad de su familia, como por ejemplo fotos con sus abuelos.
+
+También dentro de la colección, que no está digitalizada,  se pueden consultar la correspondencia del escritor como por ejemplo, las cartas que se escribía con el escritor Plinio Apuleyo Mendoza...
```
This comment is confusing to me. Are you reviewing the digital collection, or the entire collection?

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+También dentro de la colección, que no está digitalizada,  se pueden consultar la correspondencia del escritor como por ejemplo, las cartas que se escribía con el escritor Plinio Apuleyo Mendoza...
+
+
+...El archivo privado de García Márquez, muestra sin duda una “verdad” sobre su vida que no se conocía públicamente. Al revisarlo, se pueden romper con los mitos sobre su historia de vida. A través de los archivos se puede escrudiñar en sus en sus hábitos de escritura, por ejemplo. Las autocorrecciones de sus novelas, sus comunicaciones,  sus círculos de amigos íntimos, su familia, su obsesión po ...
```
typo: en sus en sus

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+Sin embargo, no se trata nada más sobre lo que el archivo pueda ofrecer, o el hecho de que está digitalizado y es accesible a todo el mundo. Tampoco se trata nada más, por ejemplo, sobre el hecho de que la colección privada de un escritor latinoamericano se encuentre en otro país lejos de la tierra donde él vivió...
+
+
+...El caso del archivo de Gabriel García Márquez también nos abre a la reflexión sobre lo que significa un archivo per se debido a la polémica que se generó cuando la familia decidió vendérselo al Harry Ransom Center.
```
Per se?

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+...El archivo significa un pasado, un presente y un futuro. Y en este sentido es comprensible que el traslado, por ejemplo, de estos archivos a otro país haya generado tanta polémica...
+
+
+...Cuando el escritor murió en el 2014 y se dió a conocer la noticia del destino de sus archivos, muchas fueron las reacciones a favor y en contra de la decisión. La idea de que el escritor les pertenece a los colombianos se diluyó al vender su memorabilia a un centro en los Estados Unidos. Yo diría que con no sólo con esta decisión García Márquez dejó de ser de los colombianos, sino que una vez  ...
```
Typos: 

con no sólo con
Todo todo
todos los que tiene

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+...También nos hace preguntarnos quién era realmente Gabriel García Márquez?  Y por qué es tan importante no sólo poseer sus archivos sino digitalizarlos y hacerlo accesibles al mundo? A quién le interesa? Cuál es su contribución para el mundo? Qué intereses políticos están detrás de los documentos y detrás del debate surgido a partir de la decisión de la familia?...
+ 
+
+...El archivo fue adquirido por el Harry Ransom Center, institución que también posee y preserva archivos de otros importantes escritores como James Joyce, Ernest Hemingway, y Jorge Luis Borges entre otros. La institución adquirió los archivos de García Márquez por 2,2 millones de dólares en noviembre del 2014 y estuvo disponible al público general en diciembre del 2017.
```
Can you link to the source for these dates?

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+
+...Cuando se discutió este tema sobre el destino de los archivos del premio Nobel muchas preguntas salieron al debate público:  hizo el gobierno colombiano lo necesario para quedarse con los archivos?  Podría invertir el gobierno o alguien en Colombia 2,2 millones de dólares para la adquisición de este material? Por qué el gobierno colombiano permitió que se llevaran esos archivos?  Si estuviese  ...
+
+Ninguna de estas preguntas tendrán respuestas, pero sí nos permiten la reflexión sobre lo que puede significar un archivo, lo que significa digitalizarlo, el significado de la accesibilidad del mismo y lo que además representa migrar los documentos a otro país , es decir, “desarraigarlo” de su entorno...
```
I'd like you to reflect a little more deeply on this question. Do you think the digital project would look different if it had been done by a colombian archive? How?

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+
+...Cuando el escritor murió en el 2014 y se dió a conocer la noticia del destino de sus archivos, muchas fueron las reacciones a favor y en contra de la decisión. La idea de que el escritor les pertenece a los colombianos se diluyó al vender su memorabilia a un centro en los Estados Unidos. Yo diría que con no sólo con esta decisión García Márquez dejó de ser de los colombianos, sino que una vez  ...
+
+Como consecuencia, el archivo se convirtió en algo etéreo y concreto a la vez...
```
The archive might feel ethereal, but that's not entirely true. For example: there are legal restrictions that explain why some of the documents were digitized, while others were not. Similarly, the digital files have a material impact (they are literally inscribed on discs) and are embedded in an equally material repository software and metadata schema. I'd like you to think more about the reality of the archive.

halperta commented on _posts/blog/2018-10-17-ReviewGarcíaMárqu:
```diff
@@ -0,0 +1,63 @@
+
+Así de extraño como el nacimiento del último de los Buendía que nació con una cola de cerdo en la famosa novela 100 años de Soledad del colombiano Gabriel García Márquez, es extraño para muchos de sus paisanos que el archivo privado de este escritor, premio Nobel de literatura 1982, reposen en el Centro de Archivos Harry Ransom Center en la Universidad de Texas en Austin, en los Estados Unidos. P ...
+
+Pero por qué la familia del escritor querría que sus archivos, los originales de sus novelas más famosas, sus notas, sus memorias, sus fotos y en fin lo que condensa parte su vida privada y que lo hizo famoso públicamente como escritor estuviese tan lejos de su macondiana tierra?...
```
The use of ellipses (...) can be an effective stylistic choice, but it should be used judiciously. Otherwise it has the effect of looking overly informal. Please choose one or two places, and remove the rest.

halperta reviewed (changes requested):
Yenibel - I enjoyed the reflective style of this piece and the way that you engage with the debate around archival ownership.

I would like you to revise, focusing on three elements:
1. Please copyedit. I have marked some typos but a thorough review is necessary.
2. Sometimes the piece is repetitive. For example, you repeat the cost of the collection several times, and return to the debate about the location of the collection multiple times as well. You can tighten the writing by reducing this repetition.
3. Instead of repeating the question of ownership, I'd like you to dig into the digital archive more deeply. If you want to think about the question of ownership more fully, think about how the fact that the archive was made in the U.S. impacts your experience as a user. Can you find evidence that the interface, metadata, framing, text, etc. has a US perspective? Similarly, what do you mean when you say "everyone has access"? Are there limits to that access? What are they?

Please submit your revision by next Friday, October 26. Let me know if you have any questions.

yenibel committed:
Update 10/29/2018 

Hi Hanna, I hope this version get better regarding formatting and content. I think that I did all the changes that you require. Let me know any new changes.

halperta committed:
Merge branch 'gh-pages' into gh-pages

halperta committed:
started copyediting

halperta reviewed (changes requested):
Yenibel - I think the writing in this revised version is much stronger, well done! However, it still has numerous typos. I started correcting them, but there were too many. If you find it difficult to review yourself, please have a colleague or friend review it with you.

Have you used the "eye" to view the document the way it will look on publication? If you do, you'll be able to see some of the errors more clearly.

yenibel commented:
Thank you. Something is happend with the typos. In my word version there is
no typos. I wonder the problem is when I copy -paste. Let me review again.

Sorry about it,
Best,
Yenibel Ruiz

On Sat, Nov 3, 2018, 9:32 AM Hannah Alpert-Abrams <notifications@github.com
wrote:

> *@halperta* requested changes on this pull request.
>
> Yenibel - I think the writing in this revised version is much stronger,
> well done! However, it still has numerous typos. I started correcting them,
> but there were too many. If you find it difficult to review yourself,
> please have a colleague or friend review it with you.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/halperta/criticalarchives/pull/15#pullrequestreview-171329581>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AqMAfiQSj56tXMtCEGdm9fGaBqJMMHkjks5urajmgaJpZM4XkMdW>
> .
>

yenibel committed:
Update 2018-11-04 Review García Máquez

I hope that now is ok. I put the text in notes and after I did copy-paste. However, you were right. There were four typos that I haven't seen before.
Saludos,
Yenibel

halperta committed:
typo in YAML

halperta committed:
adding .md and correcting typo for yenibel

halperta committed:
formatting typos in yenibel
